src/regex.js

```diff
@@ -1 +1 @@
-export default /^(?:[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}|00000000-0000-0000-0000-000000000000)$/i;
+export default /^(?:[0-9a-f]{8}-[0-9a-f]{4}-[13457][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}|00000000-0000-0000-0000-000000000000)$/i;
```

Comment:
This accepted `2` before, nice catch 👍

Reply:
I’m not sure about removal of version 2 from the regex.

v2 is clearly defined as a valid version in https://www.rfc-editor.org/rfc/rfc4122#section-4.1.3

While the RFC doesn’t provide any details how to generate such UUIDs and we don’t have support for them either, the version is still defined as valid and I think we should just stick to the RFC.

Reply:
Missed that, yeah makes sense to keep it 👍

Reply:
@ctavan @broofa as this PR is headed towards encompassing the broader RFC updates (v6 and v8 included) and likely to be merged when the RFC is ratified, I see it worthwhile to validate them as well. Any objections to the following?

```suggestion
export default /^(?:[0-9a-f]{8}-[0-9a-f]{4}-[1-8][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}|00000000-0000-0000-0000-000000000000)$/i;
```

Reply:
Yup.  Go ahead and commit.

Reply:
T